pkg/operator/vttablet/pvc.go

```diff
@@ -24,7 +24,13 @@ import (
 )
 
 // NewPVC creates a new vttablet PVC from a Spec.
+// TODO: "Handle the case when labels are removed from ExtraLabels"
```

Comment:
We don't usually put quotes around the TODO comment. Also, let's move this to right before the point where the fix would need to go, which is line 49:

```go
	// update extra labels
	// TODO: Handle the case when labels are removed from ExtraLabels.
	update.Labels(&obj.Labels, spec.ExtraLabels)
```

Reply:
moved to recommended place and removed quote